### PR TITLE
Add a prelude for Pango to export the traits

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by EPashkin** [comment]
@sdroege remove please auto/matrix.rs I added it in #65 by mistake

**Comment by sdroege** [comment]
Done

File: src/prelude.rs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2015-2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! Traits and essential types inteded for blanket imports.
+
+pub use auto::traits::*;
+
```
Comment (EPashkin) [inline src/prelude.rs]: Matrix is trait too. IMHO lib.rs need reexport PangoMatrix.

Comment (sdroege) [inline src/prelude.rs]: Ok, done :)

**Comment by EPashkin** [comment]
For this and gio remove trait=true from Gir.toml

File: src/prelude.rs (hunk 1 above)
Comment (EPashkin) [inline src/prelude.rs]: Not fully. Fix lib.rs please too.

Comment (sdroege) [inline src/prelude.rs]: What should happen with lib.rs?

Comment (sdroege) [inline src/prelude.rs]: PangoMatrix is also not re-exported from anywhere in pango currently

Comment (EPashkin) [inline src/prelude.rs]: `pub use ffi::PangoMatrix as PangoMatrix;` 
Or Matrix trait will be unusable without adding pango-sys dependence

**Comment by EPashkin** [comment]
👍 Thanks @sdroege 
@GuillaumeGomez CI finished

**Comment by GuillaumeGomez** [comment]
@sdroege: Thanks for the PR!
@EPashkin: Thanks for the review!

**Pull request merged**